artemis/src/main/java/tech/pegasys/artemis/cli/options/MetricsOptions.java

```diff
@@ -26,7 +30,7 @@
   public static final boolean DEFAULT_METRICS_ENABLED = false;
   public static final int DEFAULT_METRICS_PORT = 8008;
   public static final String DEFAULT_METRICS_INTERFACE = "127.0.0.1";
-  public static final ArrayList<String> DEFAULT_METRICS_CATEGORIES = new ArrayList<>();
+  public static final List<ArtemisMetricCategory> DEFAULT_METRICS_CATEGORIES = List.of(BEACON);
```

Comment:
I'd suggest defaulting to enabling all metrics. I hadn't noticed that we'd disabled them all by default.  

```suggestion
  public static final List<ArtemisMetricCategory> DEFAULT_METRICS_CATEGORIES = new ArrayList<>(EnumSet.allOf(ArtemisMetricCategory.class));
```

Reply:
yep ok - we're just going to use as much of the besu code as we can and try to get it all right i think